timur/lib/client/jsx/components/attributes/collection_attribute.jsx

```diff
@@ -1,40 +1,51 @@
 import MagmaLink from '../magma_link';
-import { connect } from 'react-redux';
+import {connect} from 'react-redux';
 
-import { reviseDocument } from '../../actions/magma_actions';
-import React, { Component } from 'react';
+import {reviseDocument} from '../../actions/magma_actions';
+import React, {Component} from 'react';
 import ListInput from '../inputs/list_input';
 import SlowTextInput from '../inputs/slow_text_input';
 
-const CollectionAttribute = ({ mode, value, revised_value,
-  document, template, attribute, reviseDocument }) => {
-  if (mode != 'edit') return (
-    <div className='attribute'>
-      <div className='collection'>
-        {
-          (value || []).map( link =>
-            <div key={ link } className='collection_item'>
-              <MagmaLink link={ link } model={ attribute.link_model_name }/>
+const CollectionAttribute = ({
+  mode,
+  value,
+  revised_value,
+  document,
+  template,
+  attribute,
+  reviseDocument
+}) => {
+  if (mode != 'edit') {
+    var collator = new Intl.Collator(undefined, {
+      numeric: true,
+      sensitivity: 'base'
+    });
+    return (
+      <div className='attribute'>
+        <div className='collection'>
+          {(value || []).sort(collator.compare).map((link) => (
```

Comment:
Changed to sort before mapping.

Reply:
Nit:  This sort is a mutation on the original attribute, meaning that visiting this page could change the behavior of the state implicitly.  Would it be possible to
1.  sort in the reducer somehow
2.  make a copy and then sort?

```
const sortedCollection = useMemo(() => [...value].sort(collator.compare), [value]);
```